pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2632,4 +2667,46 @@ public SchemaCompatibilityStrategy getSchemaCompatibilityStrategy() {
         }
         return schemaCompatibilityStrategy;
     }
+
+    @Deprecated
+    public void setZooKeeperSessionTimeoutMillis(long zooKeeperSessionTimeoutMillis) {
```

Comment:
Does this mean `ZooKeeperSessionTimeoutMillis` just has higher priority than `MetadataStoreSessionTimeoutMillis` when they both exists in "broker.conf" ? 

Since `getZooKeeperSessionTimeoutMillisCan` won't be used again, can we keep all logic in `getMetadataStoreSessionTimeoutMillis`, and keep the set method as default?  like 

```
public long getMetadataStoreSessionTimeoutMillis() {
  if (zooKeeperSessionTimeoutMillis > 0) return zooKeeperSessionTimeoutMillis; //it have higher priority
  return metadataStoreSessionTimeoutMillis;
}
```

Reply:
Yes, good point!

Reply:
@Jason918 Done, please help check.

Reply:
Forget remove this `setZooKeeperSessionTimeoutMillis` ?

Reply:
Oh sorry, forget this one. Removed.